com.unity.netcode.gameobjects/Runtime/NetworkVariable/NetworkVariableBase.cs

```diff
@@ -5,64 +5,113 @@ namespace Unity.Netcode
     /// <summary>
     /// Interface for network value containers
     /// </summary>
-    public interface INetworkVariable
+    public abstract class NetworkVariableBase
     {
+        /// <summary>
+        /// The name of the channel to be used for syncing
+        /// </summary>
+        public const NetworkChannel NetworkVariableChannel = NetworkChannel.NetworkVariable;
+
+        protected NetworkVariableBase() { }
+
+        protected NetworkVariableBase(NetworkVariableSettings settings)
+        {
+            Settings = settings;
+        }
+
+        // demolish me
```

Comment:
+1 for nuking it!
I took a note saying: "getting rid of NetworkObject & NetworkBehaviour deps in NetworkVariable (external read/write bools?)"